serendipity_config.inc.php

```diff
@@ -210,9 +210,6 @@
 @define('USERLEVEL_CHIEF', 1);
 @define('USERLEVEL_EDITOR', 0);
 
-@define('VIEWMODE_THREADED', 'threaded');
```

Comment:
Where did these go?

Reply:
With the wind ;)

We run into a problem here as the constants were defined both as code for internal logic and in the language constant files for the layout. I first removed the internal constants, then re-added them when seeing what was up, but changed how the logic works as well. This has to be checked again with comments display in the frontend at a later time, and possibly be corrected.

Reply:
This should then better be outsourced as a feature branch or something else - maybe at least something like a "TODO/FIXME: PHP8" line that can later be addressed?